### PR TITLE
326 create unittests for a simplified case

### PR DESCRIPTION
**Pull request by MatthiasHauthDeltares**

### CODE REVIEW

File: ra2ce/analysis/indirect/losses.py [hunk 1]
```diff
@@ -65,45 +67,67 @@ def __init__(
...
+        ):
+            raise ValueError(
+                f"Missing required columns in resilience_curve: {_required_resilience_curve_keys}"
+            )
 
     def _load_df_from_csv(
         self,
         csv_path: Path,
         columns_to_interpret: list[str],
+        sep: str = ",",
```
Comment (Carsopre) [inline ra2ce/analysis/indirect/losses.py:124]: don't add default values please, make explicit calls instead.

File: tests/analysis/indirect/test_losses.py [hunk 2]
```diff
@@ -43,58 +38,67 @@ def test_initialize(self):
...
+            / "resilience_curve.csv",
+            traffic_intensities_file=test_data
+            / "losses"
+            / "csv_data_for_losses"
+            / "traffic_intensities.csv",
+            values_of_time_file=test_data
+            / "losses"
+            / "csv_data_for_losses"
+            / "values_of_time.csv",
+            name="single_link_redundancy_losses_test",
```
Comment (Carsopre) [inline tests/analysis/indirect/test_losses.py:59]: Make your life easier:

```suggestion
        _losses_csv_data = test_data.joinpath("losses", "losses", "csv_data_for_losses")
        _analyses = AnalysisSectionIndirect(
            part_of_day=None,
            resilience_curve_file=_losses_csv_data.joinpath("resilience_curve.csv"),
            traffic_intensities_file=_losses_csv_data.joinpath("traffic_intensities.csv"),
            values_of_time_file=_losses_csv_data.joinpath("values_of_time.csv"),
            name="single_link_redundancy_losses_test",
```

File: tests/analysis/indirect/test_losses.py [hunk 3]
```diff
@@ -43,58 +38,67 @@ def test_initialize(self):
...
     @pytest.mark.parametrize(
         "part_of_day",
         [pytest.param(PartOfDayEnum.DAY), pytest.param(PartOfDayEnum.EVENING)],
     )
     def test_calc_vlh(self, part_of_day: str):
         # 1. Define test data
-        # TODO: Not sure of the input format values float of series?
         _config = AnalysisConfigWrapper()
-        _config.config_data.input_path = Path("sth")
+        _config.config_data.input_path = test_data / "losses" / "csv_data_for_losses"
```
Comment (Carsopre) [inline tests/analysis/indirect/test_losses.py:84]: Same as before.

**Review by Carsopre — CHANGES_REQUESTED** [review]
Some remarks must be processed before merging.

File: ra2ce/analysis/indirect/losses.py (hunk 1 above)
Comment (MatthiasHauthDeltares) [reply ra2ce/analysis/indirect/losses.py:124]: done

**Review by Carsopre — APPROVED** [review]

**Pull request merged**